Update  5.1 ( Skipped $banID ) (#252)
NilPointer-Software/bdfd-wiki: src/bdscript/blackListRoles.md

# $blackListRoles
Blocks users with certain role(s) from using the command. If the user has any role in the blacklist, they will not be able to run the command. Uses role names instead of role IDs.

## Syntax
```
$blackListRoles[role names;...;error message]
```

## Parameters
- `role names`: The name(s) of the role(s) to blacklist. Separate role names using `;`.

> - `error message`: The message that appears when the user has a role in the blacklist.

Reviewer: ```suggestion
- `error message`: The message that will be sent if the user has a role from the blacklist.
```
